commit 6ab7799a2fc1845d2f2da4237b5a559d6b05949e (refs/changes/98/198/4)
Author: Jonathan Perkin <jperkin@joyent.com>
Date:   2016-08-10T15:48:09+01:00 (3 years, 2 months ago)
    
    OS-5568 joyent manifest should include nis/client
    
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Patrick Mooney <patrick.mooney@joyent.com>

diff --git a/overlay/generic/usr/lib/brand/joyent/manifests b/overlay/generic/usr/lib/brand/joyent/manifests
index e147777c..bf4bb1ca 100644
--- a/overlay/generic/usr/lib/brand/joyent/manifests
+++ b/overlay/generic/usr/lib/brand/joyent/manifests
@@ -28,11 +28,9 @@
 # application/x11/x11-server.xml
 # application/x11/xfs.xml
 # application/x11/xvnc-inetd.xml
-# network/ldap/client.xml
 # network/rpc/keyserv.xml
 # network/socket-filter-kssl.xml
 # network/ssl/kssl-proxy.xml
-# network/nfs/rquota.xml
 # network/dns/server.xml
 # network/ftp.xml
 # network/finger.xml
@@ -53,10 +51,7 @@
 # network/security/ktkt_warn.xml
 # network/rpc/smserver.xml
 # network/nfs/cbd.xml
-# network/nfs/mapid.xml
-# network/nfs/status.xml
 # network/ipmievd.xml
-# network/nis/client.xml
 # network/rpc/rex.xml
 # network/http-apache22.xml
 # system/consolekit.xml
@@ -122,6 +117,7 @@ network/nfs/mapid.xml					disabled
 network/nfs/nlockmgr.xml				disabled
 network/nfs/rquota.xml					disabled
 network/nfs/status.xml					disabled
+network/nis/client.xml					disabled
 network/rexec.xml					disabled
 network/routing/legacy-routing.xml			enabled
 network/routing/ndp.xml					disabled
